xep-0440.xml

```diff
@@ -108,7 +118,7 @@
   SHOULD be the "Channel-binding unique prefix" of a channel-binding
   type which was registered with the &iana-cb-types;.</p>
 
-  <p>A server declares that it supports particular channel-binding
+  <p>A server declares that it supports a particular channel-binding
```

Comment:
"supports a … types" doesn't sound quite right, either change to "supports a … type" or keep the existing wording (assuming that it is grammatically correct).

Reply:
Oh, I don't know why I did add this stupid "a", maybe I somehow read `type` instead of `types` and then tried to fix the grammar. Dunno.

I've force-pushed a fix, thanks :)